# 清洁任务列表消息
# 包含多个清洁任务

# 任务列表
CleanTask[] tasks

# 列表时间戳
builtin_interfaces/Time timestamp

